build/cockroach.rb

```diff
@@ -150,17 +149,28 @@ def install
     ENV["GOPATH"] = buildpath
     ENV["GOHOME"] = buildpath
 
-    files = Dir.glob('*')
+    files = Dir.glob("*")
     mkdir_p buildpath/"src/github.com/cockroachdb/cockroach"
     mv files, buildpath/"src/github.com/cockroachdb/cockroach"
     Language::Go.stage_deps resources, buildpath/"src"
-    
+
     # TODO(peter): We should be using the cockroach makefile so that
     # we can get the build information baked into the
     # binary. Unfortunately, this is dying with some weird dwarf
     # linker error:
     #
     #   make -C src/github.com/cockroachdb/cockroach build SKIP_BOOTSTRAP=1
+    #
+    # Something truly bizarre is going on. The error is:
+    #
+    # # github.com/cockroachdb/c-snappy
+    # cannot load DWARF output from $WORK/github.com/cockroachdb/c-snappy/_obj//_cgo_.o: decoding dwarf section info at offset 0x0: too short
+    #
+    # But that error only happens when building with make. Using go
```

Comment:
> FYI, I'm going to fix this in a subsequent PR.

SGTM.

The env var is getting modified by https://github.com/Homebrew/homebrew/blob/8c6777b2b8e847d5f2b447eae532fa8ecffa6467/Library/ENV/4.3/make and interpreted by https://github.com/Homebrew/homebrew/blob/8c6777b2b8e847d5f2b447eae532fa8ecffa6467/Library/ENV/4.3/cc. Looks like it strips out some debug-info-related flags which means that go can't get the metadata it needs. Bizarre that that's tied to whether it's running under `make` or not.